### PR TITLE
Don't trigger a sync if HT gains focus due to a click on the project switcher #468

### LINKED ISSUE #468: Don't trigger a sync if HT gains focus due to a click on the project switcher
It's annoying when I want to switch projects but HT goes into a sync when I click on the project switcher drop down.

### PR DESCRIPTION
**Pull request by dariusf**
Fixes #468

### CODE REVIEW

File: src/ui/UI.java [hunk 1]
```diff
@@ -159,20 +160,22 @@ private void setupMainStage(Scene scene) {
 		initialiseJNA();
 		mainStage.focusedProperty().addListener(new ChangeListener<Boolean>() {
 			@Override
-			public void changed(ObservableValue<? extends Boolean> ov, Boolean was, Boolean is) {
-				if (is) {
-					
+			public void changed(ObservableValue<? extends Boolean> unused, Boolean wasFocused, Boolean isFocused) {
+				if (isFocused) {
```
Comment (damithc) [inline src/ui/UI.java]: Reverse condition reduces nesting level.
```java
if(!isFocused){
   return;
}
```

File: src/util/PlatformEx.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import javafx.application.Platform;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import service.ServiceManager;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+public class PlatformEx {
```
Comment (damithc) [inline src/util/PlatformEx.java]: A brief comment about the purpose of the class can be helpful. The class name is not self-explanatory.

**Comment by dariusf** [comment]
Okay, thanks for the comments. Should I make the changes in this branch and then merge again (under the same name)?

**Comment by damithc** [comment]
Can do in the master. No hurry.

**Pull request merged**